axelrod/strategies/alternator.py

```diff
@@ -5,7 +5,13 @@
 
 
 class Alternator(Player):
-    """A player who alternates between cooperating and defecting."""
+    """
+    A player who alternates between cooperating and defecting.
+
+    Names
+
+    - Alternator: [Bendor1993]_
```

Comment:
I'll put Axelrod's book in (it's what I did for Cooperator and Alternator). :+1: